Arrangement_on_surface_2/examples/Arrangement_on_surface_2/bgl_dual_adapter.cpp

```diff
@@ -1,10 +1,11 @@
 //! \file examples/Arrangement_on_surface_2/bgl_dual_adapter.cpp
 // Adapting the dual of an arrangement to a BGL graph.
 
+#include <CGAL/basic.h>
```

Comment:
```suggestion
#include <CGAL/config.h>
```
In my opinion `<CGAL/basic.h>` is an evil thing that includes too much of CGAL. Here `<CGAL/config.h>` should be enough.